pom.xml

```diff
@@ -296,5 +297,25 @@
                 <module>webtests</module>
             </modules>
         </profile>
+        <profile>
+            <id>doclint-java8-disable</id>
+            <activation>
+                <jdk>[1.8,)</jdk>
+            </activation>
+            <properties>
+                <javadoc.opts>-Xdoclint:none</javadoc.opts>
+            </properties>
+          <build>
+            <plugins>
+              <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-gpg-plugin</artifactId>
+                <configuration>
+                  <skip>true</skip>
```

Comment:
@vankeisb, this prevents the source from being built when running in JDK 1.8 or higher.

I tried it by using the `mvn clean install -Psonatype-oss-release` command line.